src/veileder/kandidatliste/modaler/EndreUtfallModal.tsx

```diff
@@ -31,55 +44,95 @@ const EndreUtfallModal: FunctionComponent<Props> = ({
             onRequestClose={onLukk}
             className="endreUtfallModal"
         >
-            <Systemtittel className="endreUtfallModal__tittel">
-                {utfallToDisplayName(utfall)}
-            </Systemtittel>
-            <div className="endreUtfallModal__beskrivelse">
-                <Endringsbeskrivelse
-                    utfall={utfall}
-                    kandidatnavn={`${fornavn || ''} ${etternavn || ''}`}
+            <Systemtittel className="endreUtfallModal__tittel">Endre utfall</Systemtittel>
+            <Normaltekst className="endreUtfallModal__beskrivelse">
+                Utfallet du registrerer på {navn} vil bli telt, og tellingen vil bli brukt til
+                statistikk.
+            </Normaltekst>
+            {(nyttUtfall === Utfall.Presentert || nyttUtfall === Utfall.FåttJobben) && (
+                <AlertStripeAdvarsel className="blokk-m">
+                    {alertTekst(nyttUtfall)}
+                </AlertStripeAdvarsel>
+            )}
+            <RadioGruppe legend="Velg utfall:">
+                <Radio
+                    label={
+                        <>
+                            <span>{utfallToDisplayName(Utfall.IkkePresentert)}</span>
+                            <FargetPrikk type={Utfall.IkkePresentert} />
+                        </>
+                    }
+                    value={Utfall.IkkePresentert}
+                    checked={nyttUtfall === Utfall.IkkePresentert}
+                    onChange={onEndreUtfall}
+                    name="endreUtfall"
+                />
+                <Radio
+                    label={
+                        <>
+                            <span>{utfallToDisplayName(Utfall.Presentert)}</span>
```

Comment:
I designskissen er label-teksten ikke "Presentert", men "Presentert for arbeidsgiver". Hvilken skal vi velge?

Reply:
Vi kan høre med Malin når hun er tilbake. Tenker vi kan ha det slik det er for nå? Er jo lett å endre.
Det er jo litt rart om vi noen steder refererer til Presentert som "Presentert" mens "Presentert for arbeidsgiver" andre steder...